main/main.cpp

```diff
@@ -2359,12 +2359,13 @@ bool Main::iteration() {
 	bool exit = false;
 
 	for (int iters = 0; iters < advance.physics_steps; ++iters) {
+		Engine::get_singleton()->_in_physics = true;
```

Comment:
So it can be moved out of the loop now?

Reply:
Possibly, still have to evaluate whether this should be set during the input.

Reply:
I've moved this back to the original position for now, so the agile input is flushed _outside_ the ticks, and it can no longer be moved outside the loop.

This is something we can potentially revisit.